workflows/funcscan.nf

```diff
@@ -155,15 +163,18 @@ workflow FUNCSCAN {
         if (params.arg_skip_deeparg) {
             ARG ( ch_prepped_input, [] )
         } else {
-            ARG ( ch_prepped_input, ch_annotation_output )
+            ARG ( ch_prepped_input, ch_annotation_faa )
         }
         ch_versions = ch_versions.mix(ARG.out.versions)
     }
 
     /*
         BGCs
     */
-    // TODO antismash
+    if ( params.run_bgc_screening ) {
+        BGC ( fasta_prep.compressed, ch_annotation_gff )
```

Comment:
Why not ch_prepped_input like for AMP/ARGs?

Reply:
Yes, I changed it in the upcoming gecco PR, but actually can change it here already. (I remember an error when using the unzipped FASTA, but cannot reproduce 🤷‍♀️.)